resources/js/app.js

```diff
@@ -5,21 +5,73 @@ import '@babel/polyfill';
 import store from './store';
 import {router} from './router.js';
 import {vuetify} from './plugin';
+import VueI18n from "vue-i18n";
 
 import api from './plugin/api';
 import Lodash from 'lodash';
 
 Vue.use(Lodash);
+Vue.use(VueI18n);
 
 Vue.config.productionTip = false;
 Vue.prototype.$http = api;
 
 const files = require.context('./', true, /\.vue$/i);
 files.keys().map(key => Vue.component(key.split('/').pop().split('.')[0], files(key).default));
 
+
+const i18n = new VueI18n({
```

Comment:
Misschien overzichterlijk om hier een apart bestandje voor aan te maken en die te plaatsen in plugin. Dat heb ik hetzelfde met vuetify gedaan. Die laad ik weet in met de index.js

Reply:
Heb ik verwerkt.